.github/workflows/cmake.yml

```diff
@@ -516,13 +521,23 @@ jobs:
             os: windows-latest
             cmake-args: -T ClangCl -A x64
 
+          - name: Windows ClangCl Win64 Native Instructions (AVX)
+            os: windows-latest
+            cmake-args: -T ClangCl -A x64 -DWITH_NATIVE_INSTRUCTIONS=ON -DNATIVE_ARCH_OVERRIDE="-mavx -mpclmul"
+
           - name: Windows GCC
             os: windows-latest
             compiler: gcc
             cxx-compiler: g++
             cmake-args: -G Ninja
             codecov: win64_gcc
 
+          - name: Windows GCC (AVX)
```

Comment:
**nitpick:** 
This should also have `Native Instructions` in the name.

Reply:
Fixed